types/node/globals.d.ts

```diff
@@ -1066,15 +1065,11 @@ declare namespace NodeJS {
     type TypedArray = Uint8Array | Uint8ClampedArray | Uint16Array | Uint32Array | Int8Array | Int16Array | Int32Array | Float32Array | Float64Array;
     type ArrayBufferView = TypedArray | DataView;
 
-    interface NodeRequireCache {
-        [path: string]: NodeModule;
-    }
-
     interface Require {
         /* tslint:disable-next-line:callable-types */
         (id: string): any;
         resolve: RequireResolve;
-        cache: NodeRequireCache;
+        cache: Dict<NodeModule>;
```

Comment:
This introduces "undefined" as a possible value for cache, where before it must be an object. I am not yet sure how Node.js handles this internally, but just wanted to mark a potential difference from the previous code.

Reply:
Oops! Nevermind. I misunderstood. Please ignore!